HyperV/scripts/create-environment.ps1

```diff
@@ -211,15 +212,19 @@ function cherry_pick($commit) {
 }
 
 ExecRetry {
+    ls C:\OpenStack\build\openstack\neutron
```

Comment:
ls C:\OpenStack\build\openstack  && ls C:\OpenStack\build would be useful to confirm the location / existance of folders